EventFilter/CSCRawToDigi/src/CSCTMBHeader2020_CCLUT.cc

```diff
@@ -272,18 +294,47 @@ void CSCTMBHeader2020_CCLUT::addCorrelatedLCT1(const CSCCorrelatedLCTDigi& digi)
 
 void CSCTMBHeader2020_CCLUT::addShower(const CSCShowerDigi& digi) {
   uint16_t hmt_bits = (digi.bitsInTime() & 0x3) + ((digi.bitsOutOfTime() & 0x3) << 2);
+  //not valid LCT shower, then in-time bits must be 0. keep out-of-time HMT
+  if (not digi.isValid())
+    hmt_bits = ((digi.bitsOutOfTime() & 0x3) << 2);
```

Comment:
Some tiny perf improvement could be achieved with
```suggestion
  uint16_t hmt_bits = digi.isValid()
                      ? (digi.bitsInTime() & 0x3) + ((digi.bitsOutOfTime() & 0x3) << 2)
  //if not valid LCT shower, then in-time bits must be 0. keep out-of-time HMT:
                      : ((digi.bitsOutOfTime() & 0x3) << 2);
```
(and the same in the several occurences of similar patterns in the other parts of this PR)

Reply:
fixed as suggested